### PR TITLE
Create Integration UI Changes

### PR DESCRIPTION
**Pull request by kahboom**
## Changes
- Add PatternFly colors to SCSS sheet that is easy to reference. Add reference to those colors.
- Add class to `flow-view-step.component.ts` to be able to get `active` or `inactive` statuses for parents (currently only available for submenu/child items).
- Rm icons from Steps.
- Add circular icon thingy for adding a Step (ref #262 ).
- Fix trash icon colors for active sections.
- Aesthetic changes to better reflect designs, such as padding for the sidebar items.

### Add circular icon thingy for adding a Step
screenshot 2017-03-21 19 45 55

### Aesthetic improvements 
To better reflect designs.

screenshot 2017-03-21 20 15 01

### CODE REVIEW

File: src/app/integrations/edit-page/flow-view/flow-view-step.component.html [hunk 1]
```diff
@@ -30,8 +30,11 @@ <h4 class="modal-title pull-left">Confirm Delete</h4>
 
   <div class="col-md-3 progress-line">
     <div class="icon">
-      <p [class]="'round ' + getActiveClass()">
-        <i [class]="getIconClass()"></i>
+      <p [class]="'round ' + getSubMenuActiveClass()"
+         [class.add-step-or-connection]="step.stepKind !== ('endpoint' || 'log')">
```
Comment (gashcrumb) [inline src/app/integrations/edit-page/flow-view/flow-view-step.component.html]: the `log` step doesn't need a special icon, only `endpoint` have an icon now, so maybe this check isn't needed?

File: src/app/integrations/edit-page/flow-view/flow-view-step.component.html [hunk 2]
```diff
@@ -30,8 +30,11 @@ <h4 class="modal-title pull-left">Confirm Delete</h4>
...
+        <!-- Only display icon if step kind is not an endpoint or log -->
+        <i [class]="getIconClass()"
+           *ngIf="step.stepKind === ('endpoint' || 'log')"></i>
```
Comment (gashcrumb) [inline src/app/integrations/edit-page/flow-view/flow-view-step.component.html]: Same as above

File: src/app/integrations/edit-page/flow-view/flow-view-step.component.scss [hunk 3]
```diff
@@ -86,20 +97,24 @@
         padding: 5px 0;
 
         li {
-          padding: 5px 5px 5px 17px;
+          padding: 5px 5px 5px 30px;
+
```
Comment (gashcrumb) [inline src/app/integrations/edit-page/flow-view/flow-view-step.component.scss]: Honest I'll try and stop deleting your newlines :-)

File: src/scss/_colors.scss [hunk 4]
```diff
@@ -0,0 +1,88 @@
+// PatternFly Colors
```
Comment (gashcrumb) [inline src/scss/_colors.scss]: Good call on this especially if we wind up tweaking colors to look better on bluejeans/projectors

File: src/app/integrations/edit-page/flow-view/flow-view-step.component.html (hunk 1 above)
Comment (kahboom) [inline src/app/integrations/edit-page/flow-view/flow-view-step.component.html]: Sounds good, coming to a PR near you.

File: src/app/integrations/edit-page/flow-view/flow-view-step.component.scss (hunk 3 above)
Comment (kahboom) [inline src/app/integrations/edit-page/flow-view/flow-view-step.component.scss]: rofl I can't bear it.. even in the `ts` files, but there's a lint rule for that, so it yells at me every time I try to keep the code separated and readable. :(

**Pull request merged**